richie/tests.py

```diff
@@ -0,0 +1,50 @@
+"""
+Test suite for Richie's main module
+"""
+from unittest import mock
+
+from django.test import TestCase
+from raven.exceptions import InvalidGitRepository
+
+from .settings import get_release
+
+
+# pylint: disable=unused-argument
+class GetReleaseTestCase(TestCase):
+    """Battle test the settings.get_release function"""
+
+    @mock.patch("builtins.open", side_effect=FileNotFoundError)
+    @mock.patch("raven.fetch_git_sha", side_effect=InvalidGitRepository)
+    def test_returns_default_without_version_file_or_git_repository(self, *args):
+        """
+        When not version.json file exists nor a Git repository, we expect that
```

Comment:
s/not/no ?

Reply:
Yes! Well spotted thanks. This is now fixed.